### DISCUSSION TITLE
Logging non-QSOs

### DISCUSSION THREAD

**Discussion by corrt** [magicbug/Cloudlog#849] [Q&A]
I'm ready to switch to cloudlog, but I'm looking for a way to do something specific. I want to record QSOs. but also contacts that don't count as reportable QSOs. Can I record both qsos and "qso-like" contacts, but keep them separate when time to upload to lotw or eqsl?

My specific use case is to record messages sent and received with JS8Call. But I can think of other uses: incomplete QSOs, spots, etc.

Would defining a new mode help? Or maybe, log those contacts under a different station entry? 

Thanks!

  -- Jeff, N0JUH

**Comment by magicbug** [comment]
Jeff, 

You raise an interesting idea and something I've been considering too, as it would be nice to do station notes when testing.

My thinking was kind of like a Journal that had a date/time, title, category, body and an option to upload a file or two

Theres no real point of logging these as "QSOs" IMHO especially incomplete can add to problems and would need filtering on export systems so prefer to keep the QSO table just for real stuff.

Peter.

**Comment by magicbug** [comment]
You could stick them under a specific station title, they'd still show up in the log as "worked" however which might present a problem

**Comment by corrt** [comment]
Good ideas, Peter, I guess I'll have to experiment and see what I can do
with it. I can always write a script to filter the adif, but it would be
nice to keep it separate in the logger.

From my POV, I want to know who I have exchanged messages with - when I
last heard from them - name, qth, station info. In short, all the stuff
you'd log for a QSO. But it's not a real qso per ham tradition, so I don't
want to submit it for QSLs and awards.

I see there's a mode type for WSPR. Those obviously aren't QSOs. Do they
get exported/submitted? That's why I was thinking of using my own mode type
that would be ignored or rejected by the qsl systems.

  -- Jeff, N0JUH
